core/src/main/java/org/frankframework/http/rest/ApiServiceDispatcher.java

```diff
@@ -15,44 +15,26 @@
 */
 package org.frankframework.http.rest;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Map.Entry;
-import java.util.SortedMap;
-import java.util.concurrent.ConcurrentSkipListMap;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-
-import javax.ws.rs.core.Response.Status;
-
+import jakarta.json.*;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.logging.log4j.Logger;
 import org.apache.xerces.xs.XSModel;
-import org.springframework.util.MimeType;
-
-import jakarta.json.Json;
-import jakarta.json.JsonArray;
-import jakarta.json.JsonArrayBuilder;
-import jakarta.json.JsonObject;
-import jakarta.json.JsonObjectBuilder;
-import jakarta.json.JsonValue;
 import org.frankframework.align.XmlTypeToJsonSchemaConverter;
-import org.frankframework.core.IAdapter;
-import org.frankframework.core.IPipe;
-import org.frankframework.core.ListenerException;
-import org.frankframework.core.PipeLine;
-import org.frankframework.core.PipeLineExit;
-
+import org.frankframework.core.*;
```

Comment:
Het is beter om geen wildcard imports te gebruiken. (Zie https://errorprone.info/bugpattern/WildcardImport)

Reply:
Dit is aangepast, was een IntelliJ setting